book/finances.md

```diff
@@ -314,38 +234,21 @@ slideshow:
   slide_type: ''
 tags: [remove-cell]
 ---
-# # If something was awarded, treat its weighted amount as 100% regardless of the number we had there
-# for ix, irow in leads.iterrows():
-#     if irow["% success"]:
-#         if "Awarded" in irow["Status"]:
-#             leads.loc[ix, "Amount (weighted)"] = irow["Amount for 2i2c"]
-#         else:
-#             leads.loc[ix, "Amount (weighted)"] = (
-#                 irow["Amount for 2i2c"] * irow["% success"]
-#             )
+# If we want to look at the opportunities that were dropped
+# ishow(opportunities, pageLength=50)
 ```
 
-+++ {"editable": true, "slideshow": {"slide_type": ""}}
-
-### Amortize leads across months
-
-For each lead, we spread the total amount into equal monthly amounts over the total lifetime of the contract.
-If it's a lead we use _anticipated_ start/stop/amount. If it's a contract we use the contract values.
-
 ```{code-cell} ipython3
 ---
 editable: true
 slideshow:
   slide_type: ''
-tags: [remove-cell]
+tags: [remove-input]
 ---
-# # Convert date columns to DateTime objects
-# date_cols = ["Start Date", "End Date"]
-# for col in date_cols:
-#     leads.loc[:, col] = pd.to_datetime(leads[col])
-#     # Round any dates to the nearest month start.
-#     # This controls for the fact that some dates are the 1st, others the 31st.
-#     leads.loc[:, col] = leads[col].apply(lambda x: round_to_nearest_month(x))
+# Anonymize opportunities if we are in a CI/CD environment because this will be public
+if "GITHUB_ACTION" in os.environ:
```

Comment:
I'd be tempted to make this an opt-out thing to eliminate the likelihood of this ever regressing, albeit seemingly a slim chance that GH would remove such a well-used var.